setuptools/_vendor/vendored.txt

```diff
@@ -1,3 +1,4 @@
 packaging==16.8
 pyparsing==2.2.1
 six==1.10.0
+ordered-set
```

Comment:
Shouldn't this be pinned to the particular version vendored-in?

Reply:
Perhaps. I realized that the vendored dependencies are implicitly pinned when `paver update_vendored` is run... so I'm okay with leaving this declaration unpinned. But you're right to be concerned - why the inconsistency? Just because I'd rather remove the pins... but those other dependencies have probably advanced since they were last updated. My desire is to unpin the others and run another `update` operation.

Reply:
I do think it would be valuable if the dependency isn't declared as a pinned one to capture the effectively pinned version during the `update` operation.

Reply:
On the other hand, pinning the dependencies make it easier to know which version of each package is vendored and to update only some of those dependencies.

Reply:
I've done that for now in 734d09c5.